docs/conf.py

```diff
@@ -17,20 +18,25 @@
 
 # -- General ------------------------------------------------------------------
 
+sys.path.insert(0, str(Path(__file__).parent / "_extensions"))
+
 extensions = [
     "breathe",
     "myst_parser",
     "sphinx.ext.mathjax",
+    "warnings_filter",
 ]
+
+warnings_filter_config = str(Path(__file__).parent / "known-warnings.txt")
+warnings_filter_silent = True
```

Comment:
off-pr this is where I miss some kind of export keyword in python

Reply:
True. This mechanism of `conf.py` is a bit awkward.